### DISCUSSION TITLE
image-uploader组件是否可支持自定义文件选择器？ 如果页面要在app内打开，选择附件的时候，希望唤起客户端提供的文件选择器界面，但我看image-uploader组件只有设置是否唤起照相机的配置？

### DISCUSSION THREAD

**Discussion by liangle2020** [ant-design/ant-design-mobile#5839] [Q&A]
image-uploader组件是否可支持自定义文件选择器？ 如果页面要在app内打开，选择附件的时候，希望唤起客户端提供的文件选择器界面，但我看image-uploader组件只有设置是否唤起照相机的配置？ 例如fusion mobile的附件上传组件支持自定义文件选择器，fusion mobile的文档如下
https://fusion.alibaba-inc.com/dsm/mobile/components/detail/upload?themeid=10264&tabActiveKey=component

**Comment by miracles1919** [comment]
https://github.com/ant-design/ant-design-mobile/issues/5123#issuecomment-1160148691
